core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -121,7 +121,11 @@ contract Liquidatable is PricelessPositionManager {
         uint256 liquidationId,
         bool disputeSucceeded
     );
-    event LiquidationWithdrawn(address caller, uint256 withdrawalAmount, Status liquidationStatus);
+    event LiquidationWithdrawn(
+        address indexed caller,
+        uint256 indexed withdrawalAmount,
```

Comment:
Yeah why index by this param? I can see why you'd want to index by `caller` or even `liquidationStatus`.

Reply:
Yep, exactly. Got rid of the `indexed` on this param.